llama.cpp

```diff
@@ -1840,6 +1842,7 @@ struct llama_hparams {
     uint32_t n_lora_q = 0;
     uint32_t n_lora_kv = 0;
     uint32_t n_ff_exp = 0;
+    uint32_t n_ff_shared_exp = 0;
```

Comment:
```suggestion
    uint32_t n_ff_shexp = 0;
```

Reply:
force pushed update for the name change.
diff here https://github.com/ggerganov/llama.cpp/compare/559b77819f7f6842a0dd467c5f536aeaed7433bd..46777ddaf370ec6f6b03d9a9d1797269855e4b30

Reply:
Should `n_ff_shexp` be checked here as well btw?
https://github.com/ggerganov/llama.cpp/blob/559b77819f7f6842a0dd467c5f536aeaed7433bd/llama.cpp#L1893